Orange/widgets/data/owselectbydataindex.py

```diff
@@ -45,35 +45,31 @@ def __init__(self):
         self.extra_model_unique = itemmodels.VariableListModel()
         self.extra_model_unique_with_id = itemmodels.VariableListModel()
 
-        box = gui.hBox(self.controlArea, box=None)
-        self.infoBoxData = gui.label(
-            box, self, self.data_info_text(None), box="Data")
-        self.infoBoxExtraData = gui.label(
-            box, self, self.data_info_text(None), box="Data Subset")
+        box = gui.widgetBox(self.controlArea, True)
+        gui.label(
+            box, self, """
+Even in a situation in which some or all original variables are
```

Comment:
I'd turn it around: Data rows keep their identity even when all original variables ...